package.json

```
@@ -22,28 +23,28 @@
   "scripts": {
     "tslint": "tsc --noEmit",
     "eslint": "eslint src --ext .js,.ts,.jsx,.tsx",
-    "eslint:fix": "npm run eslint -- --fix",
-    "lint": "npm run tslint && npm run eslint",
-    "lint:ts": "npm run dist",
+    "eslint:fix": "yarn eslint -- --fix",
```

Super minor: I think we can leave the `--` out

```suggestion
    "eslint:fix": "yarn eslint --fix",
```